tvtk/vtk_parser.py

```diff
@@ -641,7 +1072,8 @@ def _find_get_set_methods(self, klass, methods):
                     gsm[key] = None
 
         # Find the default and range of the values.
-        if gsm:
+        # Segfaults for some classes...
+        if gsm and klass_name not in ('vtkJavaScriptDataWriter',):
```

Comment:
Finally, we can't even init this class or it dies when garbage collected

Reply:
https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6728 should address this issue